### PR TITLE
Spotify (Independent Publisher)

### PR DESCRIPTION
**Pull request by Laskewitz**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image
image

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Woo!! Go Daniel! Thank you for submitting the Spotify connector. Can't wait for us to take a look! :)

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,943 @@
...
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Limit",
+                  "description": "The maximum number of items in the response (as set in the query or by default)."
+                },
+                "next": {
+                  "type": "string",
+                  "x-ms-summary": "Next",
+                  "description": "URL to the next page of items. ( null if none)"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Spotify/apiDefinition.swagger.json:63]: @Laskewitz Can you please remove the extra space in the rounded brackets.

`"description": "URL to the next page of items. (null if none)"`

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,943 @@
...
+                "offset": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Offset",
+                  "description": "The offset of the items returned (as set in the query or by default)"
+                },
+                "previous": {
+                  "type": "string",
+                  "x-ms-summary": "Previous",
+                  "description": "URL to the previous page of items. ( null if none)"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Spotify/apiDefinition.swagger.json:74]: @Laskewitz Can you please remove the extra space in the rounded brackets.

`"description": "URL to the next page of items. (null if none)"`

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,943 @@
...
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Limit",
+                  "description": "The maximum number of items in the response (as set in the query or by default)."
+                },
+                "next": {
+                  "type": "string",
+                  "x-ms-summary": "Next",
+                  "description": "URL to the next page of items. ( null if none)"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Spotify/apiDefinition.swagger.json:137]: @Laskewitz Can you please remove the extra space in the rounded brackets.

`"description": "URL to the next page of items. (null if none)"`

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,943 @@
...
+                "offset": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Offset",
+                  "description": "The offset of the items returned (as set in the query or by default)"
+                },
+                "previous": {
+                  "type": "string",
+                  "x-ms-summary": "Previous",
+                  "description": "URL to the previous page of items. ( null if none)"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Spotify/apiDefinition.swagger.json:148]: @Laskewitz Can you please remove the extra space in the rounded brackets.

`"description": "URL to the next page of items. (null if none)"`

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,943 @@
...
+                    "items": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/definitions/SimplifiedAlbum"
+                      },
+                      "x-ms-summary": "Albums",
+                      "description": "A list of new album releases featured in Spotify."
+                    }
+                  },
+                  "description": "A list of new album releases featured in Spotify (shown, for example, on a Spotify player's 'Browse' tab)."
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Spotify/apiDefinition.swagger.json:186]: @Laskewitz Can you please improve the description, like below:

`"description": "A list of new album releases featured in Spotify (Example: 'Browse' tab on the Spotify player)."`

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,943 @@
...
+            "required": false,
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "GetAvailableMarkets",
+              "value-collection": "Markets"
+            }
+          }
+        ],
+        "summary": "Get All New Releases",
+        "description": "Get a list of new album releases featured in Spotify (shown, for example, on a Spotify player's 'Browse' tab).",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Spotify/apiDefinition.swagger.json:207]: @Laskewitz Can you please improve the description, like below:

`"description": "A list of new album releases featured in Spotify (Example: 'Browse' tab on the Spotify player)."`

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json (hunk 2 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Spotify/apiDefinition.swagger.json:74]: Fixed.

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json (hunk 4 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Spotify/apiDefinition.swagger.json:148]: Fixed.

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json (hunk 1 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Spotify/apiDefinition.swagger.json:63]: Fixed.

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json (hunk 3 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Spotify/apiDefinition.swagger.json:137]: Fixed.

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json (hunk 5 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Spotify/apiDefinition.swagger.json:186]: Fixed.

File: independent-publisher-connectors/Spotify/apiDefinition.swagger.json (hunk 6 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Spotify/apiDefinition.swagger.json:207]: Fixed.

**Review by Laskewitz — COMMENTED** [review]
All changes are done now!

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**